cnxepub/formatters.py

```diff
@@ -249,6 +251,20 @@ def _build_binder(self, binder, elem):
 
     def build(self):
         self._build_binder(self.binder, self.body)
+        # Rewrite absolute-path links that are intra-binder
+        page_ids = [page.id for page in flatten_to_documents(self.binder)]
+        page_uuids = {id.split('@')[0]: id for id in page_ids}
+        for link in self.root.xpath('//*[@href]'):
+            href = link.get('href')
+            if href.startswith('/contents/'):
```

Comment:
Could/should inter-module links be converted so their URL is `./{UUID}` instead of `/contents/{UUID}`? It seems like that would work for online, epub, hosting-yourself-by-unzipping-the-EPUB,  and it wouldn't hardcode `/contents/` into the paths if we switch to something like `/c/`